src/ontology/fbbt-edit.obo

```diff
@@ -122779,6 +122795,23 @@ intersection_of: part_of FBbt:00001911 ! embryonic/larval nervous system
 created_by: http://orcid.org/0000-0002-6095-8718
 creation_date: 2021-06-03T18:57:00Z
 
+[Term]
+id: FBbt:00058222
+name: sugar-sensing neuron of the adult brain
```

Comment:
this is now a very generic-sounding class, when it actually refers to quite a specific cell type, I think I preferred 'protocerebrum'